debug/refresh-style.html

```diff
@@ -0,0 +1,141 @@
+    <script src="../src/Layers/BasemapLayer.js"></script>
+    <script src="../src/Layers/DynamicMapLayer.js"></script>
+
+    <script src="../src/Tasks/Task.js"></script>
+    <script src="../src/Tasks/Find.js"></script>
+    <script src="../src/Tasks/Identify.js"></script>
+    <script src="../src/Tasks/IdentifyFeatures.js"></script>
+    <script src="../src/Tasks/IdentifyImage.js"></script>
+    <script src="../src/Tasks/Query.js"></script>
+
+    <style>
+      body {
+        margin:0;
+        padding:0;
+      }
+
+      #map {
+        position: absolute;
+        top:0;
+        bottom:0;
+        right:0;left:0;
+      }
+
+      #info-pane {
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        z-index: 10;
+        padding: 1em;
+        background: white;
+      }
+      .red-div-icon {
+        background: red;
+        border: 1px solid #666;
+      }
+      .blue-div-icon {
+        background: blue;
+        border: 1px solid #666;
+      }
+    </style>
+  </head>
+  <body>
+
+  <div id="map"></div>
+  <div id="info-pane" class="leaflet-bar">
+    <label>
+    status '0' - blue icon<br>
+    status '1' - red icon<br><br>
+
+    <button onclick='postEdits()'>update 'status' attribute of objectid: 149478</button><br><br>
+    <div id='status-status'>current status of objectid: 149478 - ?</div><br>
+    <button onclick='refresh()'>refresh feature layer</button>
+    </label>
+  </div>
+  <script>
+    var bool = 0;
+
+    var map = L.map('map').setView([34.051, -117.199], 14);
+    L.esri.basemapLayer('Topographic').addTo(map);
+
+    var activeIcon = L.divIcon({className: 'blue-div-icon'});
+    var inactiveIcon = L.divIcon({className: 'red-div-icon'});
+
+    var fl = L.esri.featureLayer('http://sampleserver6.arcgisonline.com/arcgis/rest/services/RedlandsEmergencyVehicles/FeatureServer/0', {
+      pointToLayer: function (geojson, latlng) {
+        if (geojson.properties.status === 0){
+            return L.marker(latlng, {
+            icon: activeIcon
+          });
+        }
+          else {
+            return L.marker(latlng, {
+            icon: inactiveIcon
+          });
+        }
+      }
+    }).addTo(map);
+
+    function postEdits () {
+      /*
+      original:
+      X: -117.20879371400002
+      Y: 34.05838465599999
+
+      freeway-ish
+      -117.2015
+      34.0635
+      */
+
+      var updated = [{"geometry": { "x": -117.20879371400002, "y": 34.05838465599999 }, "attributes":{"OBJECTID":149478,"status":bool }}];
+
+      L.esri.post('http://sampleserver6.arcgisonline.com/arcgis/rest/services/RedlandsEmergencyVehicles/FeatureServer/0/updateFeatures', {
+          features: updated,
+          f: 'json'
+        }, function(error, response){
+        if(error){
+          console.log(error);
+        } else {
+          console.log(response);
+          bool = 1 - bool;
+          document.getElementById('status-status').innerHTML = 'current status of objectid: 149478 - ' + bool;
```

Comment:
I know this is just for debugging, but shouldn't line 129 and 130 be swapped? I was getting confusing current status info.